doc/schema/libdoc.02.xsd

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
+  <xs:element name="keywordspec" type="KeywordSpec" />
+  <xs:complexType name="KeywordSpec">
+    <xs:sequence>
+      <xs:element name="version" type="xs:string" />
+      <xs:element name="scope" type="LibraryScope" minOccurs="0" />
+      <xs:element name="namedargs" type="SupportNamedArgs" minOccurs="0" />
+      <xs:element name="doc" type="xs:string" />
+      <xs:element name="init" type="Init" minOccurs="0" />
+      <xs:element name="kw" type="Keyword" minOccurs="0" maxOccurs="unbounded" />
+    </xs:sequence>
+    <xs:attribute name="name" type="xs:string" use="required" />
+    <xs:attribute name="type" type="LibraryType" use="required" />
+    <xs:attribute name="format" type="DocumentationFormat" use="required" />
+    <xs:attribute name="scope" type="LibraryScope" use="required" />
+    <xs:attribute name="namedargs" type="xs:boolean" use="required" />
+    <xs:attribute name="generated" type="GenerationDate" use="required" />
+    <xs:attribute name="specversion" type="SpecVersion" use="required" />
+    <xs:attribute name="source" type="xs:string" use="optional" />
+    <xs:attribute name="lineno" type="xs:positiveInteger" use="optional" />
```

Comment:
There could be a comment before the above two lines about "source" and "lineno" being omitted if getting them from the library has failed for some reason.

Reply:
could you please put a comment there? You have write rights. I don't get the need for this comment, because these elements are optional, so the user have to expect that these can be not there.. for what reason ever.

Reply:
I can do that. I also agree that here on the library level a comment doesn't add so much value as source and lineno are already marked optional. On the keyword level knowing that source is omitted also if it's the same as on the library level (which is the common case) is valuable information. I also just realized we should mention that the source path is relative to the location where the spec file is created to.